api/internal/core/storage/storage_mock.go

```diff
@@ -0,0 +1,116 @@
+// Code generated by mockery v1.0.0. DO NOT EDIT.
```

Comment:
License?

Reply:
There should not be any automatically generated code in the code repository